### PR TITLE
fix(mobile): update formatting for guided vs. manual install

### PR DESCRIPTION
**Pull request by ghost**
Created from user feedback, need to point users to the guided install first.

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile-xamarin/monitor-your-xamarin-application.mdx [hunk 1]
```diff
@@ -10,6 +10,8 @@ metaDescription: "New Relic's Xamarin agent for Android and iOS: Features, compa
 
 import mobileFlutterSummary from 'images/mobile_screenshot-full_flutter-summary.webp'
 
+import mobileMobileAppToken from 'images/mobile_screenshot-crop_mobile-app-token.webp'
```
Comment (rhetoric101) [inline src/content/docs/mobile-monitoring/new-relic-mobile-xamarin/monitor-your-xamarin-application.mdx:13]: @ally-sassman I'm not seeing an instance of this image down in the body of the page. Is it possible that this image was meant for this file instead: src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/monitor-your-react-native-application.mdx

**Review by rhetoric101 — COMMENTED** [review]
@ally-sassman this project isn't building in the cloud or locally for me. Is it because of that token image that's not being used?

File: src/content/docs/mobile-monitoring/new-relic-mobile-xamarin/monitor-your-xamarin-application.mdx (hunk 1 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-xamarin/monitor-your-xamarin-application.mdx:13]:
```suggestion
```

**Comment by ally-sassman** [comment]
Fixed something after Rob's approval.

**Pull request merged**